runtime/tests/checker/resources_test.go

```diff
@@ -3571,39 +3570,82 @@ func TestCheckInvalidResourceOptionalBindingFailableCastResourceLossMissingElse(
 
 func TestCheckInvalidResourceOptionalBindingFailableCastResourceUseAfterInvalidationAfterThen(t *testing.T) {
```

Comment:
Took a moment for me to parse this test case name heh.

Reply:
Yeah, this is pretty bad. It would be nice to run `t.Run` with descriptive names for more and more tests